protonj2-client/src/main/java/org/apache/qpid/protonj2/client/impl/ClientMessage.java

```diff
@@ -61,6 +61,7 @@
 
     private int messageFormat;
 
+    // TODO there are no suppliers used in this anywhere; outdated docs?
     /**
      * Create a new {@link ClientMessage} instance with no default body section or
      * section supplier
```

Comment:
there are no suppliers used in this anywhere; outdated docs?

Reply:
Probably outdated but not really relevant to the state topic of the issue "spelling"

Reply:
Resolved in https://issues.apache.org/jira/browse/PROTON-2394, https://github.com/apache/qpid-protonj2/commit/3913c97bb8e685ddae0ab67958e96889afe4225d, I'll rebase it out of this PR.